HomeTask6/app/src/app/components/article/article.component.ts

```diff
@@ -0,0 +1,35 @@
+import { Component, OnInit } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { FormControl,FormGroup } from '@angular/forms';
+
+@Component({
+  selector: 'app-article',
+  templateUrl: './article.component.html',
+  styleUrls: ['./article.component.scss']
+})
+export class ArticleComponent implements OnInit {
+  subscription: Subscription;
+  addArticleForm = new FormGroup({
+  heading : new FormControl(''),
```

Comment:
add necessary validations here only. For example, all required fields, A date field etc

Reply:
Done